examples/react-native/src/components/Spinner.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import { ActivityIndicator, View, Dimensions } from 'react-native';
-import { connectStateResults } from 'react-instantsearch/connectors';
+import { connectStateResults } from 'react-instantsearch-core';
```

Comment:
We should import from `react-instantsearch-native`. The `core` package is only used internally.